doc/connectivity-semantics-and-api.md

```diff
@@ -48,7 +51,8 @@ also switch to IDLE to avoid connection overload at servers that are attempting
 to shed connections. We will use a default IDLE_TIMEOUT of 300 seconds (5 minutes).
 
 SHUTDOWN: This channel has started shutting down. Any new RPCs should fail
-immediately. Pending RPCs may continue running till the application cancels them.
+immediately with status UNAVAILABLE. Pending RPCs may continue running till the
```

Comment:
s/till/until/

Reply:
Doh. You noticed! I noticed when doing the "ping", but was like :-/ followed by "meh".

Done. :)